src/Layer.js

```diff
@@ -18,24 +18,23 @@ const renderers = new WeakMap();
 *
 * A Layer is the primary way to visualize geospatial data.
 *
-* To create a layer a {@link carto.source.Base|source} and {@link carto.Viz|viz} are required:
+* To create a layer a {@link carto.source|source} and {@link carto.Viz|viz} are required:
 *
-* - The {@link carto.source.Base|source} is used to know **what** data will be displayed in the Layer.
+* - The {@link carto.source|source} is used to know **what** data will be displayed in the Layer.
 * - The {@link carto.Viz|viz} is used to know **how** to draw the data in the Layer, Viz instances can only be bound to one single layer.
 *
 * @param {string} id - The ID of the layer. Can be used in the {@link addTo|addTo} function
-* @param {carto.source.Base} source - The source of the data
+* @param {carto.source} source - The source of the data
 * @param {carto.Viz} viz - The description of the visualization of the data
 *
 * @example
 * const layer = new carto.Layer('layer0', source, viz);
 *
-* @fires CartoError
```

Comment:
Why did we remove the @fires?

Reply:
After reading http://usejsdoc.org/tags-throws.html, I think we should use @throws for Error derived classes, and @fires just for events